tests/TestHelpers/WebDavHelper.php

```diff
@@ -435,6 +435,20 @@ public static function generateUUIDv4():string {
 		return vsprintf('%s%s-%s-%s-%s-%s%s%s', str_split(bin2hex($data), 4));
 	}
 
+	/**
+	 * Generates UUIDv4 with invalid characters
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+	public static function generateInvalidUUID():string {
+		// generate 16 bytes (128 bits) of random data or use the data passed into the function.
+		$data = random_bytes(16);
+		\assert(\strlen($data) == 16);
+
+		return vsprintf('%s%s-%s-%s-%s-%s%s%s', str_split($data, 2));
+	}
+
```

Comment:
don't we already have a similar function?

Reply:
i was for generateUUID